integrated_channels/integrated_channel/management/commands/transmit_courseware_data.py

```diff
@@ -0,0 +1,81 @@
+"""
+Transmits information about an enterprise's course catalog to connected IntegratedChannels
+"""
+
+from __future__ import absolute_import, unicode_literals
+
+from logging import getLogger
+
+from django.contrib.auth.models import User
+from django.core.management.base import BaseCommand, CommandError
+
+from integrated_channels.sap_success_factors.models import SAPSuccessFactorsEnterpriseCustomerConfiguration
+from integrated_channels.sap_success_factors.utils import SapCourseExporter
+
+from . import INTEGRATED_CHANNEL_CHOICES, IntegratedChannelCommandMixin, celery_task
+
+
+PLUGIN_MAPPING = {
+    SAPSuccessFactorsEnterpriseCustomerConfiguration: SapCourseExporter,
+}
+
+LOGGER = getLogger(__name__)
+
+
+class Command(IntegratedChannelCommandMixin, BaseCommand):
+    """
+    Transmit courseware data to the IntegratedChannel(s) linked to any or all EnterpriseCustomers.
+    """
+
+    def add_arguments(self, parser):
+        """
+        Add required arguments to the parser.
+        """
+        parser.add_argument(
+            '--catalog_user',
+            dest='catalog_user',
+            required=True,
+            metavar='ENTERPRISE_CATALOG_API_USERNAME',
+            help='Use this user to access the Course Catalog API. Omit this option to use the'
+            ' username specified in the EDXAPP_ENTERPRISE_CATALOG_API_USERNAME setting.'
```

Comment:
nit: please update this comment to remove the "Omit this option.." part, since it's now required.